Dockerfile.composer.no-dev

```diff
@@ -0,0 +1,12 @@
+FROM composer:1.6.3
+
+COPY composer.json \
+    composer.lock \
+    ./
+
+RUN composer --no-interaction install --no-dev --ignore-platform-reqs --no-autoloader --no-scripts --no-suggest
+
+COPY src/ src/
+
+RUN composer --no-interaction dump-autoload --no-dev --classmap-authoritative
+RUN composer --no-interaction run-script post-install-cmd
```

Comment:
We may need a base image for the `build` project images, since maintaining all the `composer` commands in every project is starting to sound onerous?

Reply:
While this Dockerfile will/should probably be identical in all PHP (micro)services. The only form of reuse seen so far is to generate it, which is automated copy and paste (e.g. `make ...` that writes it down in the project folder).

Reply:
Had wondered about extending the Composer image with our own, that contains the scripts we currently have. But don't know if it's really worth it.